### PR TITLE
trtspor.com.tr

### PR DESCRIPTION
**Pull request by gork7777**
issue: #105116

### CODE REVIEW

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -47,6 +47,7 @@
 !
 /analytics.js$domain=bataryapil.com|journey.com.tr|tozlu.com|elleshoes.com|freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|partifabrik.com|newsmax.com
 !
+||trthaber.com/service/api/stats
```
Comment (Sergey-Lyapin) [inline SpywareFilter/sections/specific.txt:50]: `||istatistik.trthaber.com^` to tracking servers firstparty please

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/specific.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply SpywareFilter/sections/specific.txt:50]: and remove this rule from specific

Comment (gork7777) [reply SpywareFilter/sections/specific.txt:50]: How do we know it's a first-party tracking server?

Comment (Sergey-Lyapin) [reply SpywareFilter/sections/specific.txt]: Because this site sends data to istatistik.trthaber.com and because it's name

**Pull request merged**